ui/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/ManagementActor.scala

```diff
@@ -125,7 +125,7 @@ class ManagementActor(managers: ProcessingTypeDataProvider[ProcessManager],
               manager.customActions.find(_.name == actionName) match {
                 case Some(customAction) =>
                   getProcessStatus(id).flatMap {
-                    case Some(status) if customAction.allowedProcessStates.contains(status.status) =>
+                    case Some(status) if customAction.allowedProcessStates.map(_.name).contains(status.status.name) =>
```

Comment:
Let's have only names in allowedProcessStates, with comment that we cannot use full state as it can contain attributes that are changing

Reply:
:+1:

Reply:
done:
https://github.com/TouK/nussknacker/pull/1379/commits/4dd43719a61b99daf936b4de63f3ad1d5c798f97

Reply:
@mproch:
https://github.com/TouK/nussknacker/pull/1379/commits/4b2f08e02fbe9ffaa793b2958828f908ef7a348c

Disable (instead of hiding) CustomActionButton when state is not loaded